jobs/system_dashboards/templates/bosh_system_disk_performance.json

```diff
@@ -970,7 +970,7 @@
         "multi": false,
         "name": "bosh_deployment",
         "options": [],
-        "query": "label_values(bosh_job_healthy{environment=~\"$environment\",bosh_name=~\"$bosh_director\"}, bosh_deployment)",
+        "query": "label_values(bosh_job_healthy{environment=~\"$environment\",bosh_name=~\"$bosh_director\",bosh_deployment!=\"bosh-health-check\"}, bosh_deployment)",
         "refresh": 1,
```

Comment:
Same as the above comment, we are NOT filtering the `compilation` job at the `bosh system` dashboards. It may be useful to diagnosticate performance problems with those vms.